docs/user/examples.rst

```diff
@@ -0,0 +1,53 @@
+Example projects
+================
+
+* Need inspiration?
+* Want to bootstrap a new documentation project?
+* Want to showcase your own solution?
+
+The following example projects show a rich variety of uses of Read the Docs. You can use them for inspiration, for learning and for recipies to start your own documentation projects.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 20 40 20
+
+   * - Topic
+     - Framework
+     - Description
+     - Links
+   * - Sphinx
+     - Sphinx
+     - Basic example of using Sphinx
+     - `[Git] <https://github.com/readthedocs-examples/example-sphinx-basic/>`__ `[Rendered] <https://example-sphinx-basic.readthedocs.io/en/latest/>`__
+   * - MkDocs
+     - MkDocs
+     - Basic example of using MkDocs
+     - `[Git] <https://github.com/readthedocs-examples/example-mkdocs-basic/>`__ `[Rendered] <https://example-mkdocs-basic.readthedocs.io/en/latest/>`__
+
+
+Real-life examples
+------------------
+
+.. image:: _static/images/awesome-list.svg
+  :alt: Awesome List badge
+  :target: https://github.com/readthedocs-examples/
+
+We maintain an **Awesome List** where you can contribute new shiny examples of using Read the Docs. Please refer to the instructions on how to submit new entries on `Awesome Read the Docs Projects <https://github.com/readthedocs-examples/>`_.
+
+
+New example projects
+--------------------
+
+We would love to add more examples that showcase documentation options. We don't want your example project to break quickly, therefore we require that your example project...
+
+* is **hosted and maintained *by you*** in its own Git repository, ``example-<topic>``
+* uses a ``.readthedocs.yaml`` configuration
+* has **continuous integration and PR builds**
+* is **versioned** as a real software project
```

Comment:
Maybe this is a recommendation instead of a requirement? Seems like a lot for an example project, and probably not obvious how it would work in general.

Reply:
Yes, agreed -- both the versioning and CI requirements are moved so they appear as recommendations.